mono/mini/mini-llvm.c

```diff
@@ -8952,6 +8955,7 @@ mono_llvm_create_aot_module (MonoAssembly *assembly, const char *global_prefix,
 	module->emit_dwarf = emit_dwarf;
 	module->static_link = static_link;
 	module->llvm_only = llvm_only;
+	module->llvm_plt_init = llvm_plt_init || llvm_only; // llvm_only implies llvm_plt_init
 	module->interp = interp;
```

Comment:
This should be the opposite, it implies !plt_init.

Reply:
Fixed to be llvm_disable_self_init